src/components/header/header.component.ts

```diff
@@ -158,9 +158,14 @@ export class DataTableHeaderComponent implements AfterViewInit {
 
     // Account for contents having style position: fixed (for scroll prevention). Calculate height necessary for cells.
     if (this.headerHeight && nativeElem.clientHeight === 0) {
```

Comment:
Here is the second change.  Everything else is just merging with the upstream repo